api/Navigator.json

```diff
@@ -3411,15 +3411,15 @@
           "spec_url": "https://w3c.github.io/badging/#setappbadge-method",
           "support": {
             "chrome": {
-              "version_added": "83",
+              "version_added": "81",
```

Comment:
Should the `clearAppBadge` data also be updated, or weren't the shipped at the same time?